src/test/java/com/wikia/webdriver/common/remote/operations/http/NoAuthOperation.java

```diff
@@ -0,0 +1,37 @@
+package com.wikia.webdriver.common.remote.operations.http;
+
+
+import com.wikia.webdriver.common.core.Helios;
+import com.wikia.webdriver.common.core.helpers.User;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.core.helpers.User'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)